router/server.go

```diff
@@ -102,6 +103,29 @@ func main() {
 	apiPort := flag.String("api-port", "", "api listen port")
 	flag.Parse()
 
+	if ports := os.Getenv("HTTP_PORTS"); ports != "" {
```

Comment:
My reasoning was remembering someone wanting to move the base ports (#3196). On further inspection, it turns out they just want to rehome sensitive applications, which doesn't require closing the base ports. So, long story short: I agree, I'll do it as additional instead (which also simplifies some logic elsewhere, I believe).